### ISSUE TITLE
Support SVG 1.1 baseline value for baseline-shift

### ISSUE THREAD

**Issue by nikosandronikos** [w3c/svgwg#246]
In [SVG 1.1, the baseline-shift](https://www.w3.org/TR/SVG/text.html#BaselineShiftProperty) property allowed the value baseline. This has been [removed in SVG 2](https://svgwg.org/svg2-draft/text.html#BaselineShiftProperty). We have some text suggesting that user agents may want to support this value, and an issue asking if we need to support it.

I'm removing the inline issue and putting this up for discussion here.
Are people ok with allowing user agents to decide if they want to support the baseline value or not?

For reference, the text of the inline issue was:
> Do we need to support the SVG 1.1 'baseline' value?

**Comment by AmeliaBR** [comment]
I would expect that any user agent that supported SVG 1.1 baseline properties would continue to support `baseline` as a deprecated synonym for `0`.

The issue is whether we should require other user agents (ahem, Firefox), who are only just getting around to supporting these properties, to do so.  

I would prefer it if they did, I don't expect it would be a huge implementation hassle to do so, but I'm open to arguments otherwise.

**Comment by dirkschulze** [comment]
Deprecating but providing a fall back is not new. If UAs support the keyword we should do this.

**Comment by dirkschulze** [comment]
It seems that the relevant CSS Inline Layout Module 3 did add `baseline` as a deprecated property value with request of UAs to comment if it can be removed. So this is solved on the CSS side.

IMO we should remove the normative text about `baseline` value from SVG2 all together. To avoid future conflicts between SVG2 and the final version of CSS Inline Layout Module 3. @AmeliaBR @Tavmjong If you don't see issues with that, I'll prepare an PR for that.

**Comment by AmeliaBR** [comment]
I agree that we should remove it from SVG 2, since [css-inline](https://drafts.csswg.org/css-inline/#baseline-shift-property) has the exact same text about supporting the value.
